packages/api-bindings/src/metadata.ts

```diff
@@ -0,0 +1,23 @@
+import { Overwrite, Prettify } from '@lens-protocol/shared-kernel';
+
+import type { PublicationMetadataV2Input } from './graphql';
+
+export type PublicationMetadata = Prettify<
+  Overwrite<PublicationMetadataV2Input, { version: '2.0.0' }>
+>;
+
+export type ProfileMetadataAttribute = {
+  displayType?: string | null;
+  key: string;
+  traitType?: string;
+  value: string;
+};
+
+export type ProfileMetadata = {
+  version: '1.0.0';
+  metadata_id: string;
+  attributes: ProfileMetadataAttribute[];
+  bio: string | null;
+  cover_picture: string | null;
```

Comment:
nit: unusual to see pascal case here over the camelcase.

Reply:
This is the metadata JSON file format, we have to use what the metadata specification tells u.